api-gateway/src/main/resources/static/scripts/bill-history/bill-history.controller.js

```diff
@@ -133,6 +133,31 @@ angular.module('billHistory')
             }
         };
 
+
+        $scope.deleteAllBills = function () {
+            let varIsConf = confirm('Are you sure you want to delete all the bills in the bill history');
```

Comment:
Looks good! Another way you could have done this is by making it so that if the user does not confirm it, it would simply return out of the function but if the user does confirm it, all the code in the function is executed. Just a small difference but both work well!

Reply:
Alright, I will try that out!

Reply:
@drallo22 I took your suggestion and have made the change so in the function it will return out of the function if the user does not confirm the delete operation

Reply:
Nice!